modules/account-lib/src/coin/cspr/utils.ts

```diff
@@ -81,6 +101,23 @@ export function isValidTransferId(id: number): boolean {
   return bigNumberAmount.isInteger() && bigNumberAmount.isGreaterThanOrEqualTo(0);
 }
 
+/**
+ * Get amount argument from deploy session
+ * Used for transfers, delegate and undelegate transactions
+ *
+ * @param {ExecutableDeployItem} transferTx transfer session
+ * @returns {string} the transfer amount
+ */
+function getAmountArgValue(transferTx: ExecutableDeployItem): string {
+  const amount = transferTx.getArgByName('amount');
+
+  if (!amount || !amount.isBigNumber()) {
+    throw new InvalidTransactionError('Transfer does not have an amount defined');
+  }
+
+  return amount.asBigNumber().toString();
```

Comment:
Can we convert using `toFixed(0)` instead?

Reply:
`asBigNumber` does not return an instance of `ethersproject` lib.
This implementation does not have a toFixed() function.

Should I use something like `new BigNumber(amount.asBigNumber().toString()).toFixed(0)`?

Reply:
hmm... I see. Your suggestion wouldn't work if `toString()` causes the number to be represented using exponential notation. Can we confirm that's not the case with this library?

Reply:
I checked and `ethersproject.BigNumber.toString()` returns the same result as `BigNumber.toFixed(0)`
So there is no issue on leaving the current implementation.